MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include languages/treesitterjavascript/src/tree_sitter/parser.h
 include languages/treesitterpython/src/tree_sitter/parser.h
+include requirements.txt
```

Comment:
Note for the future: include the entire repo. Well, except GHA automation and some helper scripts that aren't needed for running tests/building docs.

Reply:
ahhh, is that the preferred way of doing this?

Reply:
Yep. But I usually have this through `setuptools-scm`. And then, `MANIFEST.in` only has exclusions for things that for some reason aren't needed in sdists. These are usually huge blobs. Most of the time, `MANIFEST.in` doesn't need to exist, when using said plugin.

Reply:
ahhh that makes sense